pkg/scheduler/internal/cache/interface.go

```diff
@@ -56,7 +57,8 @@ import (
 // - Both "Expired" and "Deleted" are valid end states. In case of some problems, e.g. network issue,
 //   a pod might have changed its state (e.g. added and deleted) without delivering notification to the cache.
 type Cache interface {
-	framework.PodLister
```

Comment:
PodLister (returning Pods) is still used as `SharedLister#Pods()`, I suppose you will follow up with cleaning up that, right?

Reply:
Yes, likely.